automation/notebooks-table-data.csv

```diff
@@ -1,2 +1,3 @@
 display_name, notebook_name, github_repository_path, arxiv_index
 Focal Loss,losses/focal-loss.ipynb,https://github.com/facebookresearch/Detectron,1708.02002
+Pixel Shuffle, modules/pixel-shuffle.ipynb,1609.05158
```

Comment:
You forgot a comma between the path to the repository and the arxiv index

Reply:
haha, sorry